source/documentation/other-topics/migrate-to-live.html.md.erb

```diff
@@ -95,7 +97,7 @@ During the process of migration you can still deploy your app to both clusters t
 
 ## Step 2 - Amend your ingress resource
```

Comment:
@vijay-veeranki should we add this as part of header 
Amend and apply your "live-1" ingress resource